README.md

```diff
@@ -78,7 +76,7 @@ widgets.onGetAll(callbackFunction);
 
 ### Data Handler Functions
 
-The data events take a handler funciton, which takes three arguments:  `request`, `complete`, and `modules`.  `request` represents the request made to Kinvey, and `complete` is a completion handler for completing the data request.  The modules `modules` argument is an object containing several libraries for accessing Kinvey functionality via your service (for more information, see the section on [modules](#modules)).  
+The data events take a handler functions, which takes three arguments:  `request`, `complete`, and `modules`.  `request` represents the request made to Kinvey, and `complete` is a completion handler for completing the data request.  The modules `modules` argument is an object containing several libraries for accessing Kinvey functionality via your service (for more information, see the section on [modules](#modules)).
 
 #### request object
```

Comment:
Kind of a nit (and I realize it existed before this PR) but we should probably be consistent with caps in headers. Some contain caps at the first letter of all words, some at the beginning, some not at all (e.g. above)